docs/source/contributor-guide/index.md

```diff
@@ -338,3 +338,28 @@ After you've confirmed your prettier version, you can format all the `.md` files
 ```bash
 prettier -w {datafusion,datafusion-cli,datafusion-examples,dev,docs}/**/*.md
 ```
+
+## How to format `.toml` files
+
+We use `taplo` to format `.toml` files.
+
+For Rust developers, you can install it via:
+
+```sh
+$ cargo install taplo-cli --locked
```

Comment:
```suggestion
cargo install taplo-cli --locked
```

Makes sense to show prompt but can be annoying when copying from that code block (via the copy button)

Reply:
done